chapter-introduction.asciidoc

```diff
@@ -375,12 +375,12 @@ which contain a great deal of plugin customization and dependency
 declarations. But, even when your project's POM files become more
 substantial, they hold an entirely different kind of information from
 the build file of a similarly sized project using Ant. Maven POMs
-contain declarations: "This is a JAR project", and "The source code is
-in `src/main/java`". Ant build files contain explicit instructions:
-"This is project", "The source is in `src/main/java`", "Run javac
-against this directory", "Put the results in `target/classes`",
-"Create a JAR from the ....", etc. Where Ant had to be explicit about
-the process, there was something "built-in" to Maven that just knew
+contain declarations: “This is a JAR project”, and “The source code is
```

Comment:
This stuff uses curly quotes, and gets the fixed-width text inside them to format properly.